interface/src/app/analyze/analyze.js

```diff
@@ -58,9 +54,7 @@ angular.module('adage.analyze', [
       },
 
       scroll_to_id: function(id) {
-        $log.info('scroll_to_id called with: ' + id);
```

Comment:
Yeah, I noticed that this line is removed. I just want to understand the exact reason. Thanks.